lib/beetle/publisher.rb

```diff
@@ -135,6 +151,16 @@ def rpc(message_name, data, opts={}) #:nodoc:
       [status, result]
     end
 
+    def throttle(queue_options)
+      @throttling_options = queue_options
+    end
+
+    def throttle!
+      return unless throttling?
+      refresh_throttling!
+      sleep 1 if throttled?
```

Comment:
1 second might be ok for a background process (a worker or cronjob), but seems may too long for an app or api process. I suggest to make this a config option. 

Or do we suggest to better not use the throttling in app and api processes at all?

Reply:
We only want to throttle background workers/cronjobs.